src/camera/Camera.ts

```diff
@@ -129,38 +177,53 @@ abstract class Camera {
     return this;
   }
 
+  public updateAnchors(): this {
+    const flicking = getFlickingAttached(this._flicking, "Camera");
+    const panels = flicking.renderer.panels;
+
+    this._anchors = panels.map((panel, index) => new AnchorPoint({
+      index,
+      position: panel.position,
+      panel
+    }));
+
+    return this;
+  }
+
   public resetNeedPanelHistory() {
     this._needPanelTriggered = { prev: false, next: false };
   }
 
   protected _resetInternalValues() {
-    this._flicking = null;
     this._position = 0;
     this._alignPos = 0;
+    this._offset = 0;
     this._range = { min: 0, max: 0 };
     this._visiblePanels = [];
+    this._anchors = [];
     this._needPanelTriggered = { prev: false, next: false };
   }
 
   protected _refreshVisiblePanels() {
     const flicking = getFlickingAttached(this._flicking, "Camera");
     const panels = flicking.renderer.panels;
 
-    const newVisiblePanels = panels.filter(panel => panel.isVisible());
+    const newVisiblePanels = panels.filter(panel => this.canSee(panel));
```

Comment:
The verb `can` was a little awkward, but it's an appropriate naming. It also helped me read the code. :)

https://softwareengineering.stackexchange.com/questions/348649/naming-of-bool-methods-is-vs-can-vs#:~:text=According%20to%20the%20Microsoft%20naming,not%20what%20it%20can%20do.

Reply:
I named it to be natural like the case of the `if (camera.canSee(panel))` :)